### PR TITLE
Add rhsm requirement

### PR DESCRIPTION
**Pull request by lubosmj**
[noissue]

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -41,3 +41,4 @@ url-normalize>=1.4.3,<=1.4.3
 uuid6>=2023.5.2,<=2024.7.10
 whitenoise>=5.0,<6.8.0
 yarl>=1.8,<1.9.5
+rhsm>=1.19.2,<1.20.0
```
Comment (mdellweg) [inline requirements.txt:44]: Please add it as a ci_requirement. Pulpcore does not "need" in all installations.

Comment (lubosmj) [reply requirements.txt:44]: Ah, okay, thanks!

Comment (lubosmj) [reply requirements.txt:44]: We have still one outstanding issue for the lowerbounds runner. Not sure how to fix it. At least, we should have now other runners green.

**Comment by lubosmj** [comment]
Closing in favour of https://github.com/pulp/plugin_template/pull/892.

**Pull request closed without merging**